go.mod

```diff
@@ -7,12 +7,12 @@ require (
 	github.com/creachadair/jrpc2 v0.11.0
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/gammazero/workerpool v1.0.0
-	github.com/google/go-cmp v0.5.1
+	github.com/google/go-cmp v0.5.2
 	github.com/hashicorp/go-multierror v1.1.0
 	github.com/hashicorp/go-version v1.2.1
 	github.com/hashicorp/hcl-lang v0.0.0-20201116081236-948e43712a65
 	github.com/hashicorp/hcl/v2 v2.6.0
-	github.com/hashicorp/terraform-exec v0.11.1-0.20201007122305-ea2094d52cb5
+	github.com/hashicorp/terraform-exec v0.11.1-0.20201201215927-6e0fd8a457b8
```

Comment:
I'm assuming this points to a branch https://github.com/hashicorp/terraform-exec/pull/68 which means we should probably merge that PR and point to a `main` revision or a version. Not important for a WIP PR, but I'd rather avoid pointing to a revision on a branch which may disappear on a `main` of terraform-ls.

i.e. I think this should be resolved before merging this PR.